ckan/logic/__init__.py

```diff
@@ -137,6 +139,7 @@ def prettify(field_name: str):
                     assert isinstance(error, list)
                     summary[_('Tags')] = error[0]
                 else:
+                    breakpoint()
```

Comment:
Ooops..

Reply:
hehe i know, but you are looking at the older commit, already fixed in https://github.com/ckan/ckan/pull/6916/commits/1d447fddb738eaccc058fbb13a0762b5292c525b